files/rabbitmq-backup

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+DEST="$1"
+python /var/lib/rabbitmq/mnesia/*plugins-expand/rabbitmq_management-*/priv/www/cli/rabbitmqadmin export $DEST -c /etc/rabbitmqadmin.conf -N host_ssl -q
```

Comment:
Yeah, I wanted to avoid having to fetch it, but in the end it ends up being more complicated because of the globs to avoid version numbers. I've removed these scripts and now have it fetching it from https://rabbitmq.net.opencraft.hosting:15671/cli/. Thanks!